packages/report-server/examples.http

```diff
@@ -6,6 +6,12 @@
 # see: https://marketplace.visualstudio.com/items?itemName=humao.rest-client#environment-variables
 @authorization  = Basic {{email}}:{{password}}
 
+
+### Test
+GET {{baseUrl}}/test HTTP/1.1
+content-type: {{contentType}}
+Authorization: {{authorization}}
```

Comment:
Always handy to have this, as it allows us to easily check if the server is available